NumberTheory/FastPower.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+<Fast Power>
```

Comment:
Please remove the angle brackets (`<` and `>`) and indent the block by one level (4 spaces).